packages/babel-plugin-transform-nullish-coalescing-operator/src/index.js

```diff
@@ -0,0 +1,31 @@
+import syntaxNullishCoalescingOperator from "babel-plugin-syntax-nullish-coalescing-operator";
+
+export default function({ types: t }) {
+  return {
+    inherits: syntaxNullishCoalescingOperator,
+
+    visitor: {
+      LogicalExpression(path) {
+        const { node } = path;
+        if (node.operator !== "??") {
+          return;
+        }
+
+        const scope = path.scope.parent || path.scope;
```

Comment:
Still needs to be just `path.scope`. The default parameter memoization affects other transforms, and we'll need to figure out a way to deal with them all together.